bundles/org.openhab.binding.rotel/src/main/java/org/openhab/binding/rotel/internal/communication/RotelCommand.java

```diff
@@ -323,50 +544,50 @@ public enum RotelCommand {
     /**
      * Constructor when the textual commands are undefined
      *
-     * @param name the command name
+     * @param label the command label
      * @param hexType the the command type (HEX protocol)
      * @param hexKey the the command key (HEX protocol)
      */
-    private RotelCommand(String name, byte hexType, byte hexKey) {
-        this(name, hexType, hexKey, null, null);
+    private RotelCommand(String label, byte hexType, byte hexKey) {
+        this(label, hexType, hexKey, null, null);
     }
 
     /**
      * Constructor when the HEX command is undefined
      *
-     * @param name the command name
+     * @param label the command label
      * @param asciiCommandV1 the textual command (ASCII protocol V1)
      * @param asciiCommandV2 the textual command (ASCII protocol V2)
      */
-    private RotelCommand(String name, @Nullable String asciiCommandV1, @Nullable String asciiCommandV2) {
-        this(name, (byte) 0, (byte) 0, asciiCommandV1, asciiCommandV2);
+    private RotelCommand(String label, @Nullable String asciiCommandV1, @Nullable String asciiCommandV2) {
+        this(label, (byte) 0, (byte) 0, asciiCommandV1, asciiCommandV2);
     }
 
     /**
      * Constructor
      *
-     * @param name the command name
+     * @param label the command label
      * @param hexType the the command type (HEX protocol)
      * @param hexKey the the command key (HEX protocol)
      * @param asciiCommandV1 the textual command (ASCII protocol V1)
      * @param asciiCommandV2 the textual command (ASCII protocol V2)
      */
-    private RotelCommand(String name, byte hexType, byte hexKey, @Nullable String asciiCommandV1,
+    private RotelCommand(String label, byte hexType, byte hexKey, @Nullable String asciiCommandV1,
             @Nullable String asciiCommandV2) {
-        this.name = name;
+        this.label = label;
         this.hexType = hexType;
         this.hexKey = hexKey;
         this.asciiCommandV1 = asciiCommandV1;
         this.asciiCommandV2 = asciiCommandV2;
     }
 
     /**
-     * Get the command name
+     * Get the command label
      *
-     * @return the command name
+     * @return the command label
      */
-    public String getName() {
-        return name;
+    public String getLabel() {
```

Comment:
Consider overriding `toString()` also for simplified logging.

Reply:
I am taking a look...

Reply:
Done